<?xml version="1.0"?>
<launch>
    <node name="thruster_node" pkg="cbot_actuators" type="thruster_node" respawn="true" output="screen"/>
</launch>
